activesupport/lib/active_support/core_ext/date_time/calculations.rb

```diff
@@ -139,14 +146,32 @@ def end_of_minute
   end
   alias :at_end_of_minute :end_of_minute
 
-  # Adjusts DateTime to UTC by adding its offset value; offset is set to 0.
+  # Returns a <tt>Time</tt> instance of the simultaneous time in the system timezone.
```

Comment:
Just want to confirm that the data type change in the response for the 4.2.x branch is intended.

I have some test code in 4.2.7 that happens to call `.utc.rfc3339` on a `DateTime` object. On 4.2.8 this fails because `utc` converts to a `Time` object which does not support the `rfc3339` method.

I would like to clarify if this change in return type is a backwards compatibility bug for 4.2.x or perhaps `DateTime` is considered an internal-only type that client code should not work with directly?